packages/dev/core/src/Engines/nativeEngine.ts

```diff
@@ -1864,7 +1864,7 @@ export class NativeEngine extends Engine {
         const width = (<{ width: number; height: number; layers?: number }>size).width || <number>size;
         const height = (<{ width: number; height: number; layers?: number }>size).height || <number>size;
 
-        const framebuffer = this._engine.createFrameBuffer(texture._hardwareTexture!.underlyingResource, width, height, true, true);
+        const framebuffer = this._engine.createFrameBuffer(texture._hardwareTexture!.underlyingResource, width, height, true, true, 1);

```

Comment:
Why not using `options.samples || 1` instead of `1`?

Reply:
Hmm, we have a TODO above to handle options, so I ignored the options. Let me see if it'll work.